src/Libraries/DSCPython/DSCPython.csproj

```diff
@@ -9,6 +9,20 @@
     <RootNamespace>DSCPython</RootNamespace>
     <AssemblyName>DSCPython</AssemblyName>
   </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Python.Included, Version=2.5.2.7185, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Looks like this is referencing the wrong version.

Reply:
Yup, of course.